src/handlers/timecard.js

```diff
@@ -0,0 +1,99 @@
+import Boomify from 'boom';
+
+const functions = Object.create({});
+
+const goBackAYear = (date, past) => {
+
+    if ((date.getDate() <= past) && (date.getMonth() === 0)) {
+        return new Date(date.getFullYear() - 1, 11, (31 - (past - date.getDate())));
+    }
+
+    if (date.getDay() < past) {
+        const temp = new Date(date.getFullYear(), date.getMonth(), 0);
+        return new Date(date.getFullYear(), temp.getMonth(), temp.getDate() - (past - date.getDate()));
+    }
+
+    return new Date(date.getFullYear(), date.getMonth(), date.getDate() - past);
+};
+
+const lastPayPeriod = () => {
```

Comment:
Sinon doesn't support stubbing stand alone functions and I am needing to use lastPayPeriod (and goBackAYear) for testing. Have any ideas where I could put these methods to use as utility functions. Mainly because I don't want to export lastPayPeriod from timecard.js simply to use for testing.

Reply:
Also, goBackAYear could probably just be defined inside lasyPayPeriod...

Reply:
Can always create a utilities folder and put them in a file in there